src/responce.js

```diff
@@ -0,0 +1,16 @@
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+function responce(textToConvert, toCase) {
```

Comment:
It's not a response, it's just normalize
```suggestion
function makeNormalize(textToConvert, toCase) {
```